Example/Podfile.lock

```diff
@@ -5,8 +5,8 @@ PODS:
   - OCHamcrest (7.0.2)
   - OCMockito (5.0.1):
     - OCHamcrest (~> 7.0)
-  - Segment-Intercom (0.1.0):
-    - Analytics (~> 3.5)
+  - Segment-Intercom (1.0.1-beta):
```

Comment:
This beta tag seems odd?

Reply:
Looks like Dr. Sneed pushed a beta version in his last release, so that is what  is on master. Looks like previously it was 1.0.0-beta. I will remove this beta flag when I release, probably confusion regarding release process